app/src/main/java/org/dicekeys/app/views/DiceKeyCenterFaceOnlyView.kt

```diff
@@ -184,8 +184,17 @@ class DiceKeyCenterFaceOnlyView @JvmOverloads constructor(
 
     override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
         super.onMeasure(widthMeasureSpec, heightMeasureSpec)
-        val width = MeasureSpec.getSize(widthMeasureSpec).toFloat()
-        val height = MeasureSpec.getSize(heightMeasureSpec).toFloat()
+        var width = MeasureSpec.getSize(widthMeasureSpec).toFloat()
+        var height = MeasureSpec.getSize(heightMeasureSpec).toFloat()
+
+        // Find the correct width/height that will respect the aspect ratio
+        val newHeight = width * ASPECT_RATIO
+
+        if(newHeight < height){
+            height = newHeight.toFloat()
+        }else{
+            width = (height / 1.5).toFloat()
```

Comment:
Why is this not 
```
width = height / ASPECT_RATIO
```
?

Reply:
Good catch, missed it when refactoring the code.